src/assets/icons/Add/Add.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styles from './Add.module.scss';
+import classNames from 'classnames';
+const cx = classNames.bind(styles);
+
+
+interface AddProps {
+    onClick?: () => void;
+    disabled?: boolean
+}
+
+
+export const Add: React.FC<AddProps> = ({ onClick, disabled }) => {
```

Comment:
**nitpick:** We should not use `FC` - it suggests that `props.children` can be passed to this component but they are not available here